proposals/lip-0054.md

```diff
@@ -783,19 +755,19 @@ def verify(trs: Transaction) -> None:
     if trsParams.chainID == getMainchainID() or trsParams.chainID == ownChainAccount.chainID:
         raise Exception("Chain ID is not valid.")
 
-    # The commands fails if the chain is not registered.
+    # The command fails if the chain is not registered.
     if chainAccount(trsParams.chainID) does not exist:
         raise Exception("Chain is not registered.")
 
-    # The commands fails if the chain is not terminated.
+    # The command fails if the chain is not terminated.
     if terminatedStateAccount(trsParams.chainID) does not exist or terminatedStateAccount(trsParams.chainID).initialized == False:
-        raise Exception("Chain is not terminated.")
+        raise Exception("Terminated state account not present or not initialized.")
 
-    # The commands fails if there exist already a terminated outbox account.
+    # The command fails if there exist already a terminated outbox account.
     if terminatedOutboxAccount(trsParams.chainID) exists:
         raise Exception("Terminated outbox account already exists.")
 
-    queryKey = STORE_PREFIX_INTEROPERABILITY + SUBSTORE_PREFIX_CHANNEL_DATA + sha256(getMainchainID())
+    queryKey = STORE_PREFIX_INTEROPERABILITY + SUBSTORE_PREFIX_CHANNEL_DATA + sha256(OWN_CHAIN_ID)
```

Comment:
BTW: we should make sure that we have a unit test where this `verify` function is called where `OWN_CHAIN_ID != getMainchainID()`. Otherwise, I'd be a bit afraid that the implementation would not be changed as `sha256(getMainchainID())` should work for all other SKD 6 tests.

Reply:
@ishantiw

Reply:
good point, I have noted it down in the issue (https://github.com/LiskHQ/lisk-sdk/issues/8254) on SDK